test/widget/auth/email_auth_options_page_widget_test.dart

```diff
@@ -132,10 +132,11 @@ void main() {
     });
 
     testWidgets(
-        'dispatches UpdateEmailAuthOptionsPage action when tap sign in chip',
+        'dispatches UpdateEmailAuthOptionsPage action on tapping SIGN IN link',
         (WidgetTester tester) async {
       // Setup the app state with expected values
-      final initialAppState = AppState.init();
+      final initialAppState = AppState.init()
+          .rebuild((b) => b..emailAuthOptionsPage.mode = EmailAuthMode.signUp);
```

Comment:
actually this was a logic change - it makes sense to me why it passes now but I don't get why it was passing before... I'm happy to leave it as a mystery if you are

Reply:
ooh lets discuss @nickmeinhold !